src/plugins/data/common/search/expressions/kibana_context_type.ts

```diff
@@ -16,6 +17,7 @@ export type ExecutionContextSearch = {
   filters?: Filter[];
   query?: Query | Query[];
   timeRange?: TimeRange;
+  executionContext?: KibanaExecutionContext;
```

Comment:
why are we adding it to execution context, esaggs is consuming it from handlers ?

Reply:
These are leftovers of another approach I experimented with. removed.